pkg/cmd/render/render.go

```diff
@@ -107,6 +111,10 @@ func (r *renderOpts) Validate() error {
 		return errors.New("missing etcd serving CA: --manifest-etcd-serving-ca")
 	}
 
+	if err := validateBoundSATokensSigningKeys(r.generic.AssetInputDir); err != nil {
```

Comment:
Maybe set a render opt here indicating whether generation is required?

```golang
ok, err := validateBoundSATokensSigningKeys(...)
if err != nil {
  return err
}
r.BoundTokenSigningKeysProvided = ok
```

That would ensure that validation of a provided keypair could be limited to the `Validate` function, and `Run` could just respond to the option.

Reply:
passing validation does not mean the keys were provided

Reply:
but I understand your point, I think I dropped the idea of validation setting the option because it should merely validate and not provide further guidance of what to do with the provided options/files